pystiche_papers/ulyanov_et_al_2016/_loss.py

```diff
@@ -91,8 +91,8 @@ def __init__(
         self, encoder: enc.Encoder, impl_params: bool = True, **gram_op_kwargs: Any,
     ):
         super().__init__(encoder, **gram_op_kwargs)
+        # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/src/texture_loss.lua#L38
```

Comment:
As far as I see, this is the only normalization that they perform, correct? In that case, we need to create the operator with `normalize=False`. Otherwise we will be normalizing the Gram matrix by the height and width.